File: C++ programming/PROBLEMS ONC NUMBER/Program2.cpp
#include<iostream>
using namespace std;

void DisplayFactor(int iNo)
{
    if(iNo % 1 ==0)
    {cout<<"1"<<endl;}
    if(iNo % 2 ==0)
    {cout<<"2"<<endl;}
    if(iNo % 3 ==0)
    {cout<<"3"<<endl;}
    if(iNo % 4 ==0)
    {cout<<"4"<<endl;}
    if(iNo % 5 ==0)
    {cout<<"5"<<endl;}
}
int main()
{
int iValue = 0;
cout<<"Enter the Number : \n";
cin>>iValue;

DisplayFactor(iValue);
    return 0;
}

File: JAVA Programming/PROBLEMS ON NUMBER/Program2.java
import java.util.*;

class Program2
{
    public static void DisplayFactor(int iNo)
    {
        if(iNo % 1 == 0)
        {System.out.println("1");}
         if(iNo % 2 == 0)
        {System.out.println("2");}
          if(iNo % 3 == 0)
        {System.out.println("3");}
           if(iNo % 4 == 0)
        {System.out.println("4");}
            if(iNo % 5 == 0)
        {System.out.println("5");}
    }
    public static void main(String Args[])
    {   
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter First Number :");
        int iValue = sobj.nextInt();
        DisplayFactor(iValue);
    }
}